interp/tests/runt.toml

```diff
@@ -193,9 +193,9 @@ fud2 --from calyx --to jq \
 [[tests]]
 name = "numeric types correctness and parsing"
 paths = [
-  "../../tests/correctness/numeric-types/parsing/*.futil",
-  "../../tests/correctness/numeric-types/bitnum/*.futil",
-  "../../tests/correctness/numeric-types/fixed-point/*.futil",
+    "../../tests/correctness/numeric-types/parsing/*.futil",
```

Comment:
this change seems unnecessary

Reply:
I can revert it -- it's my neovim formatting setup.